docs/reference/extensibility/NuGet-Cross-Platform-Plugins.md

```diff
@@ -99,6 +102,50 @@ The plugin entry point will be the name of the installed folder, with the .dll e
 > [!Note]
 > There is currently no user story for the installation of the plugins. It's as simple as moving the required files into the predetermined location.
 
+### Support for Plugins Installed as .NET Tools
+
+Starting 6.13, NuGet now supports plugins installed as global .NET tools. This enables plugin authors to publish their plugins as .NET tools, simplifying the deployment process.
+
+#### Key Features
+
+1. Plugins installed as .NET tools must follow a naming convention: **`nuget-plugin-*`**.
+2. Upon installation, these plugins are added to the `PATH` by the .NET SDK. NuGet scans the `PATH` environment variable for executables with names starting with `nuget-plugin-`.
+3. In addition to number 1, on Windows, NuGet looks for `.exe` or `.bat` files, while on Linux and macOS, it identifies plugins by checking for the executable bit.
+4. These plugins are launched in a separate process, consistent with the existing design.

```

Comment:
I think this type of information made sense in the design, but this document is talking about all plugins period, so I think we can remove it.